--- polyr.c	2016-08-23 11:14:50.000000000 +0100
+++ polyr.c.new	2016-08-23 11:16:13.000000000 +0100
@@ -100,7 +100,7 @@
 static fract *pt3 = 0;
 
 /* Maximum degree of polynomial. */
-int MAXPOL = 0;
+static int MAXPOL = 0;
 extern int MAXPOL;
 
 /* Number of bytes (chars) in maximum size polynomial. */
@@ -111,7 +111,7 @@
  * and allocate temporary storage.
  */
 void 
-polini (int maxdeg)
+static polini (int maxdeg)
 {
 
 MAXPOL = maxdeg;
@@ -275,7 +275,7 @@
 /* c = b + a.
  */
 void 
-poladd (fract a[], int na, fract b[], int nb, fract c[])
+static poladd (fract a[], int na, fract b[], int nb, fract c[])
 {
 int i, n;
 
