--- polyr.c	2016-08-23 11:09:11.000000000 +0100
+++ polyr.c.new	2016-08-23 11:12:38.000000000 +0100
@@ -75,14 +75,14 @@
 extern void rmul ( fract *, fract *, fract * );
 extern void rdiv ( fract *, fract *, fract * );
 static void polmov ( fract *, int, fract * );
-void polmul ( fract *, int, fract *, int, fract * );
-int poldiv ( fract *, int, fract *, int, fract * );
+static void polmul ( fract *, int, fract *, int, fract * );
+static int poldiv ( fract *, int, fract *, int, fract * );
 void * malloc ( long );
 void free ( void * );
 #else
 void radd(), rsub(), rmul(), rdiv();
 static void polmov(), polmul();
-int poldiv();
+static int poldiv();
 void * malloc();
 void free ();
 #endif
@@ -145,7 +145,7 @@
 static char *form = "abcdefghijk";
 
 void 
-polprt (fract a[], int na, int d)
+static polprt (fract a[], int na, int d)
 {
 int i, j, d1;
 char *p;
@@ -201,7 +201,7 @@
 /* Set a = 0.
  */
 void 
-polclr (fract a[], int n)
+static polclr (fract a[], int n)
 {
 int i;
 
@@ -310,7 +310,7 @@
 /* c = b - a.
  */
 void 
-polsub (fract a[], int na, fract b[], int nb, fract c[])
+static polsub (fract a[], int na, fract b[], int nb, fract c[])
 {
 int i, n;
 
@@ -347,7 +347,7 @@
 /* c = b/a
  */
 int 
-poldiv (fract a[], int na, fract b[], int nb, fract c[])
+static poldiv (fract a[], int na, fract b[], int nb, fract c[])
 {
 fract *ta, *tb, *tq;
 fract quot;
@@ -506,7 +506,7 @@
 /* Evaluate polynomial a(t) at t = x.
  */
 void 
-poleva (fract a[], int na, fract *x, fract *s)
+static poleva (fract a[], int na, fract *x, fract *s)
 {
 int i;
 fract temp;
